tools/mac-distribution/internal/zipper/zipper.go

```diff
@@ -0,0 +1,63 @@
+package zipper
+
+import (
+	"archive/zip"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gravitational/trace"
+)
+
+type DirZipper interface {
```

Comment:
Do we need an interface here? Looks like there's only one implementation of it..